invisible_cities/cities/cities_test.py

```diff
@@ -60,6 +60,28 @@ def test_city_output_file_is_compressed(config_tmpdir, ICDATADIR, city):
                 continue
 
 
+@mark.filterwarnings("ignore::UserWarning")
+@mark.parametrize("city", all_cities)
+def test_city_output_contains_configuration(config_tmpdir, city):
+    file_out    = os.path.join(config_tmpdir, f"{city}_configuration.h5")
+    config_file = 'dummy invisible_cities/config/{}.conf'.format(city)
+
+    conf = configure(config_file.split())
+    conf.update(dict( file_out    = file_out
+                    , event_range = 0))
+
+    module_name   = f'invisible_cities.cities.{city}'
+    city_function = getattr(import_module(module_name), city)
+
+    city_function(**conf)
+
+    with tb.open_file(file_out) as file:
+        assert "config" in file.root
+        assert city     in file.root.config
+        table = getattr(file.root.config, city)
+        assert len(table[:]) > 0
+
```

Comment:
I really like this test and just wondering if it is easy to expand it for testing the copy. Maybe too messy to be worth it.

Reply:
Hmm, expanding it would be tricky because the input does not contain the config group, but I can make a separate test for that, at least for a single case (two consecutive cities).